package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "@groundbreaker/qewl",
-  "version": "1.0.1",
+  "version": "0.0.3",
   "description": "React Apollo and AppSync middleware designed to eliminate graqhql boilerplate code.",
   "license": "MIT",
   "author": "GroundBreaker Engineering Team <eng@groundbreaker.co>",
   "homepage": "https://github.com/groundbreaker/qewl",
   "bugs": "https://github.com/groundbreaker/qewl/issues",
   "repository": "github:groundbreaker/qewl",
   "main": "index.js",
+  "engines": {
+    "node": "~10.12.0",
```

Comment:
is this specifying the engines necessary? I find that it's a bit more trouble than its worth (in practice)